src/MinimumNodeService.cpp

```diff
@@ -149,6 +258,46 @@ Processed MinimumNodeService::handleMessage(unsigned int opc, CANFrame *msg)
 
       return PROCESSED;
 
+    case OPC_CANID:
```

Comment:
This is again something I did earlier. I have since moved it to the CanService.
Again, I believe an update from main branch will fix this.